modules/core/03-connection/keeper/keeper.go

```diff
@@ -208,6 +208,25 @@ func (k Keeper) GetAllConnections(ctx sdk.Context) (connections []types.Identifi
 	return connections
 }
 
+// GetSentinelLocalhostConnectionEnd returns the sentinel localhost connection end.
+func (k Keeper) GetSentinelLocalhostConnectionEnd() types.ConnectionEnd {
```

Comment:
Maybe also a bit of a nit, but should we rename this to `GenerateSentinelLocalhostConnection` or `CreateSentinelLocalhostConnection` because we usually use the verb `Get` for methods that retrieve data from the store. I would also remove the suffix `End` to keep it consistent with the naming in the rest of the keeper.

Reply:
Yes this sounds like a good change!